src/Dapr.AspNetCore/DaprEndpointRouteBuilderExtensions.cs

```diff
@@ -78,5 +77,67 @@ public static IEndpointConventionBuilder MapSubscribeHandler(this IEndpointRoute
                 await writer.FlushAsync();
             });
         }
+
+        /// <summary>
+        /// Maps an endpoint that will respond to requests to <c>/dapr/subscribe</c> from the
+        /// Dapr runtime.
+        /// </summary>
+        /// <param name="endpoints">The <see cref="IEndpointRouteBuilder" />.</param>
+        /// <param name="options">Configuration options</param>
+        /// <returns>The <see cref="IEndpointConventionBuilder" />.</returns>
+        /// <seealso cref="MapSubscribeHandler(IEndpointRouteBuilder)"/>
+        public static IEndpointConventionBuilder MapSubscribeHandler(this IEndpointRouteBuilder endpoints, SubscribeOptions options)
```

Comment:
This duplicates all of the existing functionality. Can you make the existing method call into this one?

Reply:
Refactored code.